tests/test_manifest.py

```diff
@@ -9,6 +9,11 @@
 from smartsim.exp.ray import RayCluster
 from smartsim.settings import RunSettings
 
+try:
+    import tensorflow
+except ImportError:
+    pass
```

Comment:
This is actually just a workaround for a clash happening when importing Ray before TensorFlow. This way, we make sure TF is imported before Ray. 

Maybe we can add a single-line comment, as I understand it can be confusing for reviewers (I suggested this workaround, that's why I'm explaining this :) )

Reply:
Ahh that makes perfect sense!! In that case disregard original feedback and a single line comment to the nature of `# import tf before ray` should be more than sufficient for future readers

Reply:
This is quite cryptic without a comment, so I'll go ahead and add that!